### ISSUE TITLE
question: local time-machine and automatic push to external drive when plugged in

### ISSUE THREAD

**Issue by bbigras** [digint/btrbk#368]
Is possible to use btrbk to have local time-machine style snapshots and also push them to an external drive automatically when available?

Or do people use btrbk for local time-machine snapshots and use a script to push them to an external drive from time to time?

Thanks

**Comment by digint** [comment]
> Is possible to use btrbk to have local time-machine style snapshots and also push them to an external drive automatically when available?

Yes, you can e.g. add a script calling btrbk on your automounter scripts, systemd triggers, ...

> Or do people use btrbk for local time-machine snapshots and use a script to push them to an external drive from time to time?

This is really up to you, btrbk has no boundaries here. You can run btrbk from cron jobs / systemd timers, manually or wherever you want. Combine this with actions (e.g. run "btrbk snapshot" and "btrbk resume" in separate scrips) and filters (e.g. run "btrbk resume /mnt/my_backup_disk").
